dwaveoceansdk/package_info.py

```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '4.2.0'
-__author__ = 'D-Wave Systems Inc.'
+# This file is kept for backwards compatibility
```

Comment:
Is it worth noting deprecation in this file so users don't add code that relies on it?

Reply:
IMO there is no harm keeping it for now, but totally fine to add a deprecation warning if we think we should.

Part of my reasoning for not bothering is I don't think we're ultimately going to keep the `dwaveoceansdk` namespace at all.